lab_2_tokenize_by_bpe/main.py

```diff
@@ -47,6 +79,22 @@ def merge_tokens(
     :param pair: a pair of tokens to be merged
     :return: dictionary in the form of <preprocessed word: number of occurrences>
     """
+    if not (isinstance(word_frequencies, dict)
+            and isinstance(pair, tuple)):
+        return None
+    new_freq_dict = {}
+    for word in word_frequencies.keys():
+        if ''.join(pair) in ''.join(word):
+            list_word = list(word)
+            for ind in range(len(word) - 1):
```

Comment:
Congratulations on solving the secret! But there are some mistakes. Think about this line. It is in this place that you have problems and you get "ве" instead of "ле". Maybe debugging can be useful for you! Good luck!